keyboards/xdboards/recon/recon.c

```diff
@@ -0,0 +1,102 @@
+/* Copyright 2024 XD Boards <copyright@xdboards.xyz>
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef OLED_ENABLE
+oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
+   const oled_rotation_t user_rotation = oled_init_user(rotation);
+   if (!user_rotation) {
+       return OLED_ROTATION_180;
+   }
+   return user_rotation;
```

Comment:
User rotation is handled a bit weirdly, and doesn't need to be called here. 
```suggestion
   return OLED_ROTATION_180;
```

For reference, from the oled driver code: 
https://github.com/qmk/qmk_firmware/blob/2ddfd60be70c12dcd09cdecb1170396ad267e17d/drivers/oled/oled_driver.c